_i18n/es.yml

```diff
@@ -341,3 +341,10 @@ latest-publications:
   posts: Últimos posts
 author:
   publications-title: Publicaciones
+thanks:
+  get-back-soon: Agradecemos que se haya tomado el tiempo de escribirnos. Nos pondremos en contacto contigo pronto.
+  our-blog: Nuestros posts
+  video-section: Sección de videos
+  services-we-offer: Servicios que ofrecemos
+  until-then: >
+    Hasta entonces, puedes consultar algunos de nuestros recursos:
```

Comment:
It sounds better for me "Nos pondremos en contacto contigo pronto" becase we specify to who we are going to contact (contigo).
"Nos pondremos en contacto contigo lo antes posible" works for me as well.
Any other opinion on that ? @alvarogarcia7 @guillemf